src/rules.jl

```diff
@@ -244,6 +244,8 @@ _abs_deriv(x) = signbit(x) ? -one(x) : one(x)
 @define_diffrule LogExpFunctions.log1mexp(x) = :(-exp($x - LogExpFunctions.log1mexp($x)))
 @define_diffrule LogExpFunctions.log2mexp(x) = :(-exp($x - LogExpFunctions.log2mexp($x)))
 @define_diffrule LogExpFunctions.logexpm1(x) = :(exp($x - LogExpFunctions.logexpm1($x)))
+@define_diffrule LogExpFunctions.log1pmx(x) = :(-$x / (1 + $x))
+@define_diffrule LogExpFunctions.logmxp1(x) = :(inv($x) - 1)
```

Comment:
Just curious, any particular reason to use this instead of
```suggestion
@define_diffrule LogExpFunctions.logmxp1(x) = :((1 - $x) / $x)
```
which would be more similar to the line above?

Reply:
I don't know if any of them would be preferable. Are there floating-point concerns?

Reply:
Honestly, I don't know. I was just surprised by the two different approaches in this PR and became curious. I used `(1 - x) / x` for the ChainRules definitions in LogExpFunctions now. In general, sometimes a reason to avoid `inv(x)` would be undesired promotions but in this case here it doesn't matter (as far as I can see).

Reply:
I will use the same as in ChainRules then for consistency